### PR TITLE
Added Setting Loading Screen as Main Scene Feature + Refactoring

### PR DESCRIPTION
**Pull request by jedouard98**
Summary:

- Added a function to make the loading screen the main scene automatically.
- Refactored some path variable names that were confusing (are they files or directories?). 
- Renamed the verbose function names in LoadingScreenGenerator and AssetBundleVerifierWindow

### CODE REVIEW

**Comment by jedouard98** [comment]
As discussed offline, added dialog box to ask user if they want to change their main scene in build. 

Sample:

image

File: GooglePlayInstant/Editor/QuickDeploy/LoadingScreenGenerator.cs [hunk 1]
```diff
@@ -68,46 +71,64 @@ public static void GenerateLoadingScreenScene(string assetBundleUrl, string load
...
                 Debug.LogWarning(warningMessage);
-                
-                DialogHelper.DisplayMessage(LoadingScreenSaveErrorTitle, warningMessage);
+
+                DialogHelper.DisplayMessage(SaveErrorTitle, warningMessage);
             }
+            else
+            {
+                //TODO: investigate GUI Layout errors that occur when moving this to DialogHelper
+                if (EditorUtility.DisplayDialog("Change Main Scene in Build?",
```
Comment (thanwah) [inline GooglePlayInstant/Editor/QuickDeploy/LoadingScreenGenerator.cs]: "Change Scenes in Build"

File: GooglePlayInstant/Editor/QuickDeploy/LoadingScreenGenerator.cs [hunk 2]
```diff
@@ -68,46 +71,64 @@ public static void GenerateLoadingScreenScene(string assetBundleUrl, string load
...
+                    "Would you like to change the main scene in build to the loading screen?", "Change", "Do Not Change"))
```
Comment (thanwah) [inline GooglePlayInstant/Editor/QuickDeploy/LoadingScreenGenerator.cs]: "Would you like to replace any existing Scenes in Build with the loading screen scene?"

I'm ok with "Change"/"Do Not Change", but slightly prefer Yes/No.

**Review by thanwah — APPROVED** [review]
LGTM.

**Comment by jedouard98** [comment]
In response to the comments, above, the new dialog: 

image

**Pull request merged**